package-examples/function-export/README.md

```diff
@@ -0,0 +1,9 @@
+# Function Export Example
+
+The blueprint is part of Kpt Function Export Guide:
+
+- `resources/resources.yaml`: declares a `Deployment` and a `Namespace`.
+- `resources/constraints/`: declares constraints used by the `gatekeeper-validate` function.
+- `functions.yaml`: runs two functions from [Kpt Functions Catalog](../../catalog) declaratively:
+  - `gatekeeper-validate` enforces constraints over all resources.
+  - `label-namespace` adds a label to all Namespaces.
```

Comment:
Let's put the order of functions as

 - `label-namespace` adds a label to all Namespaces.
 - `gatekeeper-validate` enforces constraints over all resources.

Since `label-namespace` is a transformer and `gatekeeper-validate` is a validator. The transformer should be before validator.